src/hotspot/share/runtime/os.cpp

```diff
@@ -2127,7 +2127,7 @@ char* os::attempt_reserve_memory_between(char* min, char* max, size_t bytes, siz
     assert(is_aligned(result, alignment), "alignment invalid (" ERRFMT ")", ERRFMTARGS);
     log_trace(os, map)(ERRFMT, ERRFMTARGS);
     log_debug(os, map)("successfully attached at " PTR_FORMAT, p2i(result));
-    MemTracker::record_virtual_memory_reserve((address)result, bytes, CALLER_PC);
+    MemTracker::record_virtual_memory_reserve((address)result, bytes, CALLER_PC, mtNone);
```

Comment:
I think attempt_reserve_memory_between should provide the correct mem tag.

Reply:
I wasn't sure what that is here, we can do this in a follow up?

Reply:
My suggestion was to change `attempt_reserve_memory_between` to take MemTag as an argument. It's OK to do that as a follow-up PR.

Reply:
I decided to do this now, since I had to go through many files anyhow.